.github/workflows/netgen.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2021 Sai Charan Lanka <lankasaicharan123@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'netgen'
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 0 * * 5'
+  workflow_dispatch:
+  repository_dispatch:
+    types: [ netgen ]
+
+env:
+  DOCKER_BUILDKIT: 1
+  DISPLAY: 0

```

Comment:
Is this necessary? Tools are not tested locally, but executed inside containers. Therefore, this envvar should not be useful.